inc/saved-links/class-saved-links.php

```diff
@@ -172,12 +172,37 @@ public static function display_custom_fields() {
 	<input type='text' name='lr_source' value='<?php echo $link_source; ?>' style='width:98%;'/></p>
 
 	<?php if( $link_img_src ) { ?>
-		<p><label><?php _e( 'Import featured image:', 'link-roundups' ); ?></label><br />
+		<p><label><?php _e( 'Import Featured Image:', 'link-roundups' ); ?></label><br />
 		<img src="<?php echo $link_img_src ?>" width="300" />
 		<input type='hidden' name='argo_link_img_url' value='<?php echo $link_img_src; ?>'/><br>
-		<input type="checkbox" value="1" name="lr_img" id="lr_img"><label for="lr_img"><?php _e( 'Import as feature image', 'link-roundups' ); ?></label>
+		<input type="checkbox" value="1" name="lr_img" id="lr_img"><label for="lr_img"><?php _e( 'Use social media image?', 'link-roundups' ); ?></label>
```

Comment:
We have "Import Featured Image:" above, thought that it was redundant to include again so attempting to provide context of image source but see the confusion. We can nix. Could simply "Import?"